decidim-blogs/config/locales/en.yml

```diff
@@ -3,7 +3,9 @@ en:
   activemodel:
     attributes:
       post:
```

Comment:
It's probably not in the scope of this PR but it seems we have a bug here with the `:body` attribute.

This seems to apply to the translated fields as this code does not work if you try to run it in the IRB console:
```ruby
Decidim::Blogs::Admin::PostForm.new.human_attribute_name(:body)
```

Relevant lines:
https://github.com/decidim/decidim/blob/2176c9238a1be37da3caa3ea904e6880fa9ee9c4/decidim-core/lib/decidim/form_builder.rb#L530-L536

Reply:
That gives me the `method_missing: undefined method human_attribute_name` exception. 

Maybe I don't fully understand the problem, but as far as I see, `human_attribute_name` is a class method and not an instance method, right? 

```ruby
Decidim::Blogs::Admin::PostForm.human_attribute_name(:body)
```

Reply:
Right! I guess I wasn't thinking properly.

What I meant to say that I think the `body` is missing from those attribute locales and even if I try to add it there, this field is not changing.

But I noticed this translation is actually coming from `decidim.blogs.admin.models.components.body` which seems a bit weird when the other attributes are coming from `activemodel.attributes.post.XYZ`.

It's fine for this PR as this is only fixing the missing strings/issues with strings but maybe something we should look into?

Reply:
Ah OK, I thought there was something else that I was missing!! That's fixed on #10092